### ISSUE TITLE
error while scanning cloudfront

### ISSUE THREAD

**Issue by amitvocate** [nccgroup/ScoutSuite#1409]
while running the scan, i get the following error:

`   2022-03-21 12:43:18 ip-172-16-1-105.ec2.internal scout[22483] ERROR browser.py L107: Unable to get "view_certificate" from target object {...}`

i can also see that in the target object there is a parameter called `viewer_certificate` and not `view_certificate`, so i think this is the issue here.

- The **exact** CLI parameters used to run Scout Suite:
`python scout.py aws -r us-east-1`
- Any specific configuration within the cloud account which might have lead to the issue:
We have cloudfront with certificate.

**Comment by ShahidAkhterSec** [comment]
is it fixed? Still showing error to me

**Comment by lapolis** [comment]
same here

**Comment by gnanakeethan** [comment]
It looks like a typo for me. 

"view_certificate" instead of "viewer_certificate",

I also stumbled upon this issue.

**Comment by DasMeDevon** [comment]
Confirm I'm still seeing this with 5.11.0. A quick local hotfix to check for this condition added @ https://github.com/nccgroup/ScoutSuite/blob/master/ScoutSuite/providers/base/configs/browser.py#L106

```python
 elif p == 'view_certificate':
                    target_obj = target_obj['viewer_certificate']
```

**Comment by drarnold** [comment]
Submitted MR to resolve these scanning errors
https://github.com/nccgroup/ScoutSuite/pull/1450

**Comment by fernando-gallego** [comment]
Solved in version 5.12.0
